mailpoet/tests/acceptance/Forms/EditorAddDividerCest.php

```diff
@@ -53,8 +53,10 @@ public function addDividerBlock(\AcceptanceTester $i) {
     $i->wantTo('Check divider on front end');
     $postUrl = $i->createPost('Title', 'Content');
     $i->amOnUrl($postUrl);
-    $i->assertAttributeContains('[data-automation-id="form_divider"]', 'style', 'height: 10px;');
-    $i->assertAttributeContains('[data-automation-id="form_divider"]', 'style', 'width: 10%;');
-    $i->assertAttributeContains('[data-automation-id="form_divider"]', 'style', 'border-top: 10px dotted black;');
+    $i->assertCssProperty('[data-automation-id="form_divider"]', 'height', '10px');
+    $i->assertAttributeContains('[data-automation-id="form_divider"]', 'style', 'width: 10%');
```

Comment:
Is this just forgotten here or is there a valid reason not to use `$i-> assertCssProperty('[data-automation-id="form_divider"]', 'width', '10%');`?

Reply:
It is not forgotten, I forgot to add a note on the PR. Thanks for noticing.

Because the value is computed when using `assertCssProperty`, the percentage becomes a number that depends on the theme/other elements on the page.
It didn't seem essential on the test, as we test it in the editor and we test the other properties on the front end, so I removed it.
Let me know what you think.

Reply:
Oh, right, I see. It's not removed though, it stayed there in the original `assertAttributeContains` form in the diff but as it's probably property that doesn't get normalized by Chrome (except for the space that is there anyway), then I guess it works on both Chrome and Chromium and it's fine.